app/services/katello/pulp3/smart_proxy_repository.rb

```diff
@@ -82,6 +82,21 @@ def orphan_repositories
         end
         repo_map
       end
+
+      def current_repositories_data(environment = nil, content_view = nil)
```

Comment:
This is not needed right now since the smart_proxy_service is always pulp2 here, not sure if or what to do about it?

https://github.com/Katello/katello/blob/3e3dc6dae8c2b0c01e62ab8cf6db935a920a430c/app/models/katello/concerns/smart_proxy_extensions.rb#L349